api/v1alpha1/flowcollector_types.go

```diff
@@ -45,6 +45,9 @@ type FlowCollectorSpec struct {
 
 	// CNO contains settings related to the cluster network operator
 	CNO ClusterNetworkOperator `json:"cno,omitempty"`
+
+	// Operators contains compatible operators names to install, comma separated E.g. "loki-operator,strimzi-kafka-operator,grafana-operator"
+	Operators string `json:"op,omitempty"`
```

Comment:
yes sure, we can also add some extra config per operator like versions / endpoints / ports & so on ?

Reply:
In the meanwhile, maybe, instead of a comma-separated string, I'd use a string array.

Maybe I'd also use a more descriptive field name for the JSON/Yaml: `operators`? `dependencies`?

Reply:
Maybe add a separate bool field for each operator (BTW: what are the defaults ?) 
If we deploy a stand-alone Loki, grafana etc. instance in network-observability the default should be probably to install the those in the namespace

Reply:
will be managed by separated jsons
https://github.com/netobserv/network-observability-operator/pull/59#discussion_r814643994